Frends.ServiceBus.Read/Frends.ServiceBus.Read/Read.cs

```diff
@@ -0,0 +1,271 @@
+using System.Text;
+using Microsoft.Azure.ServiceBus.Management;
+using Microsoft.Azure.ServiceBus.InteropExtensions;
+using System.Net.Mime;
+using System.Linq;
+
+namespace Frends.ServiceBus.Read;
+
+/// <summary>
+/// Azure Service Bus task.
+/// </summary>
+public class ServiceBus
+{
+    /// <summary>
+    /// Read message from Azure Service Bus queue or topic.
+    /// </summary>
+    /// <param name="input">Input parameters</param>
+    /// <param name="options">Options parameters</param>
+    /// <param name="cancellationToken">Token generated by Frends to stop this task.</param>
+    /// <returns>Object { bool ReceivedMessage, string Content, string ContentType, string SessionId, string MessageId, string CorrelationId, int DeliveryCount, long EnqueuedSequenceNumber, long SequenceNumber, string Label, Dictionary&lt;string, object&gt; Properties }, string ReplyTo, string ReplyToSessionId, long Size, string To, DateTime ScheduledEnqueueTimeUtc</returns>
+    public static async Task<Result> Read([PropertyTab] Input input, [PropertyTab] Options options, CancellationToken cancellationToken)
+    {
+        cancellationToken.ThrowIfCancellationRequested();
+
+        if (options.CreateQueueOrTopicIfItDoesNotExist)
+        {
+            var deleteIdle = TimeSpan.Zero;
+
+            if (options.AutoDeleteOnIdle > 0)
+            {
+                switch (options.TimeFormat)
+                {
+                    case TimeFormat.Minutes:
+                        deleteIdle = options.AutoDeleteOnIdle > 5 ? TimeSpan.FromMinutes(options.AutoDeleteOnIdle) : TimeSpan.FromMinutes(5); ; break;
+                    case TimeFormat.Hours:
+                        deleteIdle = options.AutoDeleteOnIdle > 0.0833333333 ? TimeSpan.FromHours(options.AutoDeleteOnIdle) : TimeSpan.FromMinutes(5); break;
+                    case TimeFormat.Days:
+                        deleteIdle = options.AutoDeleteOnIdle > 0.00347222222 ?  TimeSpan.FromDays(options.AutoDeleteOnIdle) : TimeSpan.FromMinutes(5) ; break;
+                }
+            }
+
+            switch (input.SourceType)
+            {
+                case QueueOrTopic.Queue:
+                    if (string.IsNullOrWhiteSpace(input.QueueOrTopicName) || string.IsNullOrWhiteSpace(input.ConnectionString)) throw new Exception("Connection parameters required.");
+                    await EnsureQueueExists(input.QueueOrTopicName, input.ConnectionString, deleteIdle, options.MaxSize, cancellationToken);
+                    break;
+                case QueueOrTopic.Topic:
+                    if (string.IsNullOrWhiteSpace(input.QueueOrTopicName) || string.IsNullOrWhiteSpace(input.SubscriptionName) || string.IsNullOrWhiteSpace(input.ConnectionString)) throw new Exception("Connection parameters required.");
+                    await EnsureTopicExists(input.ConnectionString, input.QueueOrTopicName, input.SubscriptionName, deleteIdle, options.MaxSize, cancellationToken);
+                    break;
```

Comment:
IMO also these should be put to multiple lines.

Reply:
I think it's cleaner to keep these in 1 line because you can word wrap these lines in Visual Studio when using smaller resolution.